README.md

```diff
@@ -211,6 +211,15 @@ More useful information following soon.
 
 Useful information about possible configurations following soon.<br>
 
+#### Github Integration
+
+To ensure your GitHub project always has the latest version of the UDF --
+
+1. Open your prefered shell (cmd, powershell, bash, zsh)
+2. Navigate to your GitHub Autoit repository
+3. Run `git submodule add https://github.com/Danp2/au3WebDriver`
```

Comment:
Keep in mind that submodules do need updating, because otherwise it will be stuck on the same version forever. I think its worth pointing out on how to update.

Reply:
@micwoj92 Good point. Can you share the required steps so that they can be included?

Reply:
@rcmaehl Is there any way to use this method to update to the latest official release instead of just grabbing the latest source from `master`?

Reply:
`git submodule update --remote --merge` to update submodule, then `git add` that path, commit and finally push as you would do with normal changes. I suspect if one is using github desktop (or for that sake any gui/tui client) application that workflow can look different.

https://git-scm.com/docs/git-submodule

As for pointing to latest release, I don't know if you can specify git tag, but you can always grab commit of the tag there.

Generally it just works like a repo within a repo, if you want to update submodule then you update the submodule repo first and then add, commit, push these changes to the parent repo.

Reply:
I don't think I explained it well... there are hundreads better blogposts on internet on how use it (if you need practical examples).